libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -70,6 +72,18 @@ export class FieldsService {
     q: new FullTextSearchField(),
     license: new LicenseSearchField(this.injector),
     owner: new OwnerSearchField(this.injector),
+    producer: new MultilingualSearchField(
+      'originatorOrgForResourceObject',
+      this.injector,
+      'asc',
+      'key'
+    ),
+    publisher: new MultilingualSearchField(
+      'distributorOrgForResourceObject',
+      this.injector,
+      'asc',
+      'key'
+    ),
```

Comment:
I remember @jahow proposing `producerOrg` and `publisherOrg` as names to make it more obvious these come from Org objects as well. I personally don't have a strong opinion on this.